Source/WebCore/html/DOMTokenList.h

```diff
@@ -30,13 +30,13 @@
 
 namespace WebCore {
 
-class DOMTokenList {
+class DOMTokenList final {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     using IsSupportedTokenFunction = Function<bool(Document&, StringView)>;
     DOMTokenList(Element&, const QualifiedName& attributeName, IsSupportedTokenFunction&& isSupportedToken = { });
 
-    void associatedAttributeValueChanged(const AtomString&);
+    inline void associatedAttributeValueChanged(const AtomString&);
```

Comment:
`inline` is not needed here AFAIK.

Reply:
Oh, I thought it would be a good idea to document it in the case someone moves the function to a *Inlines.h file in the future.